wkcuber/README.md

```diff
@@ -1,155 +1,49 @@
-# WEBKNOSSOS cuber (wkcuber)
-[![PyPI version](https://img.shields.io/pypi/v/wkcuber)](https://pypi.python.org/pypi/wkcuber)
-[![Supported Python Versions](https://img.shields.io/pypi/pyversions/wkcuber.svg)](https://pypi.python.org/pypi/wkcuber)
```

Comment:
I would keep the little icons/shields and update them to wk-libs or copy from that readme